webapp/src/components/singlestudy/explore/Modelization/Areas/index.tsx

```diff
@@ -1,8 +1,83 @@
-import UnderConstruction from "../../../../common/page/UnderConstruction";
-import previewImage from "./preview.png";
+/* eslint-disable react-hooks/exhaustive-deps */
+import { Box } from "@mui/material";
+import * as R from "ramda";
+import { ReactNode, useMemo } from "react";
+import { useOutletContext } from "react-router";
+import { StudyMetadata } from "../../../../../common/types";
+import SimpleLoader from "../../../../common/loaders/SimpleLoader";
+import NoContent from "../../../../common/page/NoContent";
+import SplitLayoutView from "../../../../common/SplitLayoutView";
+import AreaPropsView from "./AreaPropsView";
+import AreasTab from "./AreasTab";
+import useStudyData from "../../hooks/useStudyData";
+import { getCurrentAreaId } from "../../../../../redux/selectors";
+import useAppSelector from "../../../../../redux/hooks/useAppSelector";
+import useAppDispatch from "../../../../../redux/hooks/useAppDispatch";
+import { setCurrentArea } from "../../../../../redux/ducks/studydata";
 
 function Areas() {
-  return <UnderConstruction previewImage={previewImage} />;
+  const { study } = useOutletContext<{ study?: StudyMetadata }>();
+  const {
+    value: areas,
+    error,
+    isLoading,
+  } = useStudyData({
+    studyId: study ? study.id : "",
+    selector: (state) => state.areas,
+  });
+  const currentArea = useAppSelector(getCurrentAreaId);
+  const dispatch = useAppDispatch();
+  const selectedArea = useMemo(() => {
+    if (areas !== undefined && currentArea) {
+      return areas[currentArea];
+    }
+    return undefined;
+  }, [currentArea, areas]);
+
+  const handleAreaClick = (areaName: string): void => {
+    if (areas === undefined) return;
+    const elm = areas[areaName.toLowerCase()];
+    if (elm) {
+      dispatch(setCurrentArea(areaName.toLowerCase()));
+    }
+  };
+
+  return (
+    <SplitLayoutView
+      left={
+        <Box width="100%" height="100%">
+          {areas !== undefined && !isLoading && (
+            <AreaPropsView
+              areas={Object.keys(areas).map((item) => areas[item])}
```

Comment:
Don't give `areas` prop, instead use `const areas = useAppSelector(getStudyAreas);` inside `AreaPropsView` component. 

The selector:
 ```javascript
 const getStudyAreas = createSelector(
  getStudyData, 
  (studyData) => studyData ? Object.values(studyData.areas) : []
);
 ```